components/base/Table.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { ROBOTO } from "../typography";
+import { HEADLINE_TEXT, PARAGRAPH_TEXT, GREY_1, GREY_2 } from "../colors";
+
+// Styles for Table
+const StyledTable = styled.table`
+  table {
+    /* Separate borders to selectively apply border radius */
+    border-collapse: separate;
+    border-spacing: 0;
+    text-align: left;
+    font-family: ${ROBOTO};
+    overflow-x: scroll;
```

Comment:
Should we include some styles here that **A)** allow the table's width to expand to the width of its parent, and **B)** set some kind of minimum width (here or at the `td` level to prevent the cells from scrunching up on narrower screens before the overflow kicks in?

Reply:
That's a good idea! I'll take care of those 👍